src/tests/AvoInspector.test.ts

```diff
@@ -142,60 +136,92 @@ describe("Initialization", () => {
     expect(inspector.environment).toBe(AvoInspectorEnv.Staging);
   });
 
-  test("Version", () => {
+  test("Prod env is set using AvoInspectorEnv", () => {
     // When
     let inspector = new AvoInspector({
       apiKey: "apiKey",
       env: AvoInspectorEnv.Prod,
-      version: "1",
+      version: "0",
     });
 
     // Then
-    expect(inspector.version).toBe("1");
-  });
-
-  // XXX TODO does not compile, make a .js test against this behavior
-  // test("Undefined version", () => {
-  //   try {
-  //     new AvoInspector("api key", AvoInspectorEnv.Prod, undefined);
-  //     throw Error(
-  //       "Avo Inspector should throw an error if no version is provided"
-  //     );
-  //   } catch (e) {
-  //     expect(e.message).toMatch(
-  //       "[Avo Inspector] No version provided. Many features of Inspector rely on versioning. Please provide comparable string version, i.e. integer or semantic."
-  //     );
-  //   }
-  // });
-
-  test("Empty version", () => {
-    try {
+    expect(inspector.environment).toBe(AvoInspectorEnv.Prod);
+  });
+
+  test("Prod environment is set using string", () => {
+    // When
+    let inspector = new AvoInspector({
+      apiKey: "apiKey",
+      env: "prod",
+      version: "0",
+    });
+
+    // Then
+    expect(inspector.environment).toBe(AvoInspectorEnv.Prod);
+  });
+
+  // TODO: does not compile, make a .js test against this behavior
+  // TODO: is other environmets supported
+  test("Other environment is set", () => {
+    // When
+    const env = "test";
+
+    let inspector = new AvoInspector({
+      apiKey: "apiKey",
+      version: "0",
+      // @ts-ignore
+      env,
+    });
+
+    // Then
+    expect(inspector.environment).toBe(env);
+  });
+
+  test("Version is set", () => {
+    const version = "1";
+
+    // When
+    let inspector = new AvoInspector({
+      apiKey: "apiKey",
+      env: AvoInspectorEnv.Prod,
+      version,
+    });
+
+    // Then
+    expect(inspector.version).toBe(version);
+  });
+
+  // TODO: does not compile, make a .js test against this behavior
```

Comment:
Does it?

Reply:
Or you uncommented all of the non compiling tests to actually migrate them to js?

Reply:
The test compiles as `@ts-ignore` is used. Uncommented so that test cases are run and the functional implementation for the test case can be checked, but left the comment to maybe think about better approach and check this with you guys

Reply:
I see, that's good.